index.html

```diff
@@ -1,231 +1,298 @@
 
-                    <li class="social-icon">
-                        <img src="images/m.png" alt="m">
-                    </li>
-                </ul>
-                <div class="button-resume">
-                    <button type="button" class="btn">
-                        Get my resume
-                    </button>
-                </div>
             </div>
-            <div class="right">
-                <div class="lang">
+            <ul class="language-list">
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="images/arrowdown.png" alt="arrow">
+                </li>
+                <li class="lang-skill">
+                    <img src="images/js.png" alt="js">
+                    <p>
+                        JavaScript
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="images/css.png" alt="html">
+                    <p>
+                        CSS
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="images/html.png" alt="css">
+                    <p>
+                        HTML
+                    </p>
+                </li>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="images/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="images/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <!--Add contact section-->
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me</h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing,
+                a feature
+                that you need built or a project that needs coding. I’d love
+                to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
```

Comment:
- [ ] Please decrease the `margin-top ` and add a bit of `margin-left` into the contact form button to adjust the placement of the button to make it look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=7jqj8aDna8hNS8yv-0) as the aim is to make your design look as close to the figma design as possible. 

Figma design:

<img width="303" alt="Screenshot 2023-04-05 at 10 40 26 AM" src="https://user-images.githubusercontent.com/44798044/229991871-3d4f3816-e15d-48c4-a987-db43268905d0.png">

Your design:

<img width="366" alt="Screenshot 2023-04-05 at 10 39 53 AM" src="https://user-images.githubusercontent.com/44798044/229992102-b0e4cb37-bf63-4a3a-8714-2177d9c5a387.png">